chat-plugins/poll.js

```diff
@@ -78,20 +80,17 @@ var Poll = (function () {
 	};
 
 	Poll.prototype.update = function () {
-		var results = this.generateResults();
-
 		// Update the poll results for everyone that has voted
 		for (var i in this.room.users) {
 			var user = this.room.users[i];
 			if (this.voters.has(user.latestIp)) {
-				user.sendTo(this.room, '|uhtmlchange|poll' + this.room.pollNumber +  '|' + results);
+				user.sendTo(this.room, '|uhtmlchange|poll' + this.room.pollNumber +  '|' + this.generateResults(false, user));
```

Comment:
Moving `generateResults` inside an inner loop sounds laggy. There are ways to do it that aren't laggy. Who's the one who can't code again? ;)